src/guides/v2.4/install-gde/system-requirements-tech.md

```diff
@@ -47,6 +47,9 @@ Magento only uses MySQL features compatible with MariaDB. MariaDB may not be com
 
 While Magento supports PHP 7.3, Magento 2.4 is tested on, and we recommend using, PHP 7.4.
 
+{.bs-callout-warning}
+There is a [bug](https://bugs.php.net/bug.php?id=79174) in PHP 7.4.2 which causes some messages to not render properly. Magento recommends using other versions of 7.4.
```

Comment:
Should we just say that Magento does not support 7.4.2?

Reply:
I am reticent to put that right below the line that says we support 7.4. Pointing them to the bug shows it's not us and it's been fixed.